beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -225,13 +310,64 @@ def normalize(tensor):
 plt.imshow(fbank.t().numpy(), cmap='gray')
 
 
+######################################################################
+# You can create mel frequency cepstral coefficients from a raw audio signal
+# This matches the input/output of Kaldi’s compute-mfcc-feats.
+# 
+
+mfcc = torchaudio.compliance.kaldi.mfcc(waveform, **params)
+
+print("Shape of mfcc: {}".format(mfcc.size()))
+
+plt.figure()
+plt.imshow(mfcc.t().numpy(), cmap='gray')
+
+
+######################################################################
+# Available Datasets
+# -----------------
+# 
+# If you do not want to create your own dataset to train your model, ``torchaudio`` offers a
+# unified dataset interface. This interface supports lazy-loading of files to memory, download 
+# and extract functions, and datasets to build models.
+# 
+# The datasets ``torchaudio`` currently supports are:
+#
+# -  **VCTK**: Speech data uttered by 109 native speakers of English with various accents
+#    (`Read more here <https://homepages.inf.ed.ac.uk/jyamagis/page3/page58/page58.html>`_).
+# -  **Yesno**: Sixty recordings of one individual saying yes or no in Hebrew; each
+#    recording is eight words long (`Read more here <https://www.openslr.org/1/>`_).
+# -  **Common Voice**: An open source, multi-language dataset of voices that anyone can use
+#    to train speech-enabled applications (`Read more here <https://voice.mozilla.org/en/datasets>`_).
+# -  **LibriSpeech**: Large-scale (1000 hours) corpus of read English speech (`Read more here <http://www.openslr.org/12>`_).
+# 
+
+yesno_data = torchaudio.datasets.YESNO('~/datasets/YESNO/', download=True)
+
+# The data in YESNO gives us the output tuple (image, target) where target is index of the target class.
+
+print ("Image: {}".format(yesno_data[0]))
+plt.figure()
+plt.plot(yesno_data[0][0].t().numpy())
```

Comment:
How about?
```
# Pick data point number 3
n = 3
waveform, sample_rate, labels = yesno_data[n]

print("Waveform: {}\nSample rate: {}\nLabels: {}".format(waveform, sample_rate, labels))

plt.figure()
plt.plot(waveform.t().numpy())
```

Reply:
I don't think yesno data has sample_rate, output is:
```
(tensor([[ 3.0518e-05,  6.1035e-05,  3.0518e-05,  ..., -2.7466e-03,
          -3.6926e-03, -1.6174e-03]]),
 ['0', '0', '0', '1', '0', '0', '0', '1'])
```
I changed this to 
`waveform, labels = yesno_data[n]`

let me know what you think!

Reply:
Since there was a change in dataset interface, make sure you are compiling from source, see [code](https://github.com/pytorch/audio/blob/master/torchaudio/datasets/yesno.py#L20) :)

Reply:
AH! My mistake, got it!